src/collections/initiatives/recM2F4PCtF3ufqZ3.md

```diff
@@ -1,17 +1,21 @@
 ---
 title: Accessible Surveys
 id: recM2F4PCtF3ufqZ3
-permalink: '/initiatives/recM2F4PCtF3ufqZ3/'
-eventDate: '2020-08-04'
-shortDescription: "A\_conversation on\_accessible survey platforms, hosted by the Inclusive Design Research Centre. Join the discussion with David Berman, Roland Van Oostveen, Pina D'Intino, Julia Foster, Sarah Stadder Wise, Michelle Borgal, and members of the inclusive design community.\n\nAugust 4, 2020, 2:00 – 3:30 PM (EST)"
-previewImageUrl: >-
-  https://dl.airtable.com/.attachments/1f23e46ecca1375046369ab8229ddb0a/e7d8f19f/group82.jpg
-previewImageAltText: ''
-coverImageUrl: >-
-  https://dl.airtable.com/.attachments/f5d282fcac73b5359a4562c21c560c9b/88950a55/AccessibleSurveyPlatforms_1024x536.png
-coverImageAltText: >-
-  Online Conversation on Accessible Survey Platforms. Tuesday, August 4th. 2:00
-  PM - 3:30 PM. Register for free
+permalink: /initiatives/recM2F4PCtF3ufqZ3/
+eventDate: 2020-08-04T03:00:00.000Z
+coverImageUrl: https://dl.airtable.com/.attachments/f5d282fcac73b5359a4562c21c560c9b/88950a55/AccessibleSurveyPlatforms_1024x536.png
```

Comment:
Shall we download images from Airtable and place in a local directory? When we clean up Airtable, these images won't be accidentally removed.

Reply:
Oh yeah, good call. I'll do that.